app/routes/accounts.$id._index.tsx

```diff
@@ -0,0 +1,195 @@
+    error: undefined,
+    account: accountRes,
+    timeline: timelineRes,
+    loggedInAccountID: loggedInAccountDatum.response.id,
+  };
+};
+
+export const meta: MetaFunction<typeof loader> = ({ data }) => {
+  if (!data) {
+    return [{ title: "Account | Caramel" }, { content: "noindex" }];
+  }
+  if (data.error !== undefined) {
+    return [{ title: "Account | Caramel" }, { content: "noindex" }];
+  }
+
+  return [
+    { title: `${data.account.nickname} (${data.account.name}) | Caramel` },
+    { content: "noindex" },
+  ];
+};
+
+export default function Account() {
+  const data = useLoaderData<typeof loader>();
+  if (data.error !== undefined) {
+    return <div>{data.error}</div>;
+  }
+
+  const timelineNotes = data.timeline.map(
+    (note): NoteProps => ({
+      id: note.id,
+      content: note.content,
+      contentsWarningComment: note.contents_warning_comment,
+      author: {
+        avatar: note.author.avatar,
+        name: note.author.name,
+        nickname: note.author.display_name,
+      },
+      reactions: note.reactions.map((reaction) => ({
+        emoji: reaction.emoji,
+        reactedBy: reaction.reacted_by,
+      })),
+      loggedInAccountID: data.loggedInAccountID ?? "",
+    })
+  );
+  const params = useParams();
+  const isThisAccountSelf = !!params.id && params.id === loggedInAccount?.name;
+
+  return (
+    <>
+      <div className={styles.accountData}>
+        <div>
+          <img
+            className={styles.avatarImage}
+            src={data.account.avatar}
+            alt={`${data.account.nickname}'s avatar image`}
+            loading={"lazy"}
+          />
+        </div>
+        <h1>
+          {data.account.nickname} <span>{data.account.name}</span>
+        </h1>
+        <p>{data.account.bio}</p>
+        <div>
+          <p>
+            <a href={`/accounts/${data.account.name}/following`}>
+              {data.account.following_count} following
+            </a>
+            <a href={`/accounts/${data.account.name}/follower`}>
+              {data.account.followed_count} followers
+            </a>
+          </p>
+        </div>
+      </div>
+
+      {timelineNotes.length === 0 ? (
+        // NOTE: paramsで指定されたアカウントが空である場合は404が表示されるので，ここではnon-null assertionを使う
+        <EmptyAccountTimeline
+          accountName={params.id!}
+          isThisAccountSelf={isThisAccountSelf}
+        />
+      ) : (
+        <AccountTimeline notes={timelineNotes} />
+      )}
+    </>
+  );
+}
+
+interface AccountTimelineProps {
+  notes: readonly NoteProps[];
+}
+
+const AccountTimeline = ({ notes }: AccountTimelineProps) => {
+  return (
+    <div>
+      <div>
+        <LoadMoreNoteButton type="newer" noteID={notes[0].id} />
+      </div>
+
+      {notes.map((note) => {
+        return <Note key={note.id} {...note} />;
+      })}
+
+      <div>
+        {notes.length < 20 ? (
+          <></>
+        ) : (
+          <LoadMoreNoteButton type="older" noteID={notes.at(-1)!.id} />
+        )}
```

Comment:
```suggestion
        {notes.length >= 20 && (
          <LoadMoreNoteButton type="older" noteID={notes.at(-1)!.id} />
        )}
```
`&&` を使って条件が真のとき反映されるように書いたほうがシンプルになるかと思います.